### DISCUSSION TITLE
Secure boot support in bios with no secure boot support

### DISCUSSION THREAD

**Discussion by Pegoku** [ventoy/Ventoy#1682] [Q&A]
Hello and I would like to know if I can enable secure boot support in ventoy 1.0.77 and use it in a bios without secure boot support
Thank you, I would like you to answer asap

**Comment by Tech-Gamer** [comment]
Yes, Ventoy will work if the BIOS/UEFI does not have secure boot enabled and secure boot is enabled in Ventoy.
You should see this message:
![image](https://user-images.githubusercontent.com/64653625/175750981-8cfebbad-55f5-4732-9cf9-b8ad5f1c1379.png)
And after a few seconds Ventoy will load.
